manifests/filter/ip_user.pp

```diff
@@ -0,0 +1,102 @@
+# == Resource: repose::filter::ip_user
+#
+# This is a resource for generating ip identity configuration files
+#
+# === Parameters
+#
+# [*ensure*]
+# Bool.  Ensure config file present/absent
+# Defaults to <tt>present</tt>
+#
+# [*filename*]
+# String.  Config filename
+# Defaults to <tt>ip-user.cfg.xml</tt>
+#
+# [*quality*]
+# List of filters by name, which conains a list of headers
```

Comment:
Is the description for this parameter correct.  It says that it should be a list of filters, but the default is a float.